src/main.rs

```diff
@@ -86,3 +86,45 @@ fn print_verbose_err() {
          https://github.com/rust-lang/rust-bindgen/issues/new"
     );
 }
+
+#[cfg(test)]
+mod test {
+    use std::path::PathBuf;
+
+    fn build_flags_output_helper(builder: &bindgen::Builder) {
+        let mut command_line_flags = builder.command_line_flags();
+        command_line_flags.insert(0, "bindgen".to_string());
+
+        let flags_quoted: Vec<String> = command_line_flags
+            .iter()
+            .map(|x| format!("'{}'", x))
```

Comment:
Why is this quoting needed?

Reply:
Patterns (or filenames) could have spaces. To be fully correct, I would add `shlex` quoting, but I didn't feel like adding the dependency.

If you would prefer, I could use `shlex` (it's already a dev-dependency).

Reply:
Ah, yeah, I had missed you were using this only for printing... Is this really worth it compared to `println!("{:?}", command_line_flags)`?